src/backend/catalog/dependency.c

```diff
@@ -198,6 +200,8 @@ static const Oid object_classes[] = {
 	TransformRelationId,		/* OCLASS_TRANSFORM */
 
 	/* GPDB additions */
+	ProfileRelationId,		/* OCLASS_PROFILE */
+	PasswordHistoryRelationId,	/* OCLASS_PASSWORDHISTORY */
```

Comment:
Why add new object class at this position? Perhaps we should add it at the end of the array.